src/components/app-top-bar.js

```diff
@@ -187,10 +221,15 @@ const AppTopBar = ({ user, tabIndex, onChangeTab, userManager }) => {
                 onSearchTermChange={searchMatchingEquipments}
                 onSelectionChange={showVoltageLevelDiagram}
                 elementsFound={equipmentsFound}
-                renderElement={renderEquipmentForSearchBar(
-                    equipmentClasses,
-                    intl
-                )}
+                renderElement={(props) => {
+                    return (
```

Comment:
remove this return here for simplification and switch '{' to '('

Reply:
done